static/app/components/createAlertButton.tsx

```diff
@@ -441,9 +446,7 @@ const StyledCode = styled('code')`
 
 const StyledCloseButton = styled(Button)`
```

Comment:
Kind of looks like we can just kill this component?

Reply:
oh true! done in https://github.com/getsentry/sentry/pull/32781/commits/d0022f9a26621d7d4709ef742bb28128d8fd2f31